app/importers/tariff_rate/morocco_data.rb

```diff
@@ -0,0 +1,9 @@
+module TariffRate
+  class MoroccoData
+    include ::Importer
+    include TariffRate::Importer
+
+    self.default_endpoint = 'FTA_Morocco_Data.csv'
+    self.country_code = 'MA'
```

Comment:
Since tariff rates sources are by country, is there any reason why we need this `country_code` attribute in addition to the model's `source[:code]` attribute?